etc/hub.bash_completion.sh

```diff
@@ -366,7 +366,21 @@ EOF
     fi
   }
 
+  __hub_main() {
+    local can_unalias
+
+    if [ "$(type -t git)" != alias ]; then
+      alias git=hub
+      can_unalias=true
+    fi
+
+    __git_main "$@"
+
+    if [ -n "$can_unalias" ]; then
+      unalias git
```

Comment:
Neat workaround! We should watch out for the potential pitfalls of creating a temporary alias (e.g. would it maybe affect some other shell mechanism?) but for now, I can see this working. Let me try this out and report back.